content/en/news/releases/1.15.x/announcing-1.15.5/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Istio 1.15.5
+linktitle: 1.15.5
+subtitle: Patch Release
+description: Istio 1.15.5 patch release.
+publishdate: 2023-1-30
+release: 1.15.5
+aliases:
+    - /news/announcing-1.15.5
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.15.4 and Istio 1.15.5.
+
+{{< relnote >}}
+
+# Changes
+
+- **Added** the `--revision` flag to `istioctl analyze` making it possible to specify a revision.
+  ([Issue #38148](https://github.com/istio/istio/issues/38148))
+
+- **Fixed** a bug causing the abnormal exit of istiod when DestinationRule `PortLevelSettings[].Port` was nil. ([Issue #42598](https://github.com/istio/istio/issues/42598))
+
+- **Fixed** a bug causing namespace level network labels (`topology.istio.io/network`) to take precedence over workload level ones. ([Issue #42675](https://github.com/istio/istio/issues/42675))
```

Comment:
should we mention https://github.com/istio/istio/pull/42968 as well?

Reply:
The text for this seems to mention workload while I think a similar item in the 1.16.2 RN mention pod.

Reply:
Thanks! LGTM... leaving it to @ericvn to merge!